app_dart/lib/src/service/branch_service.dart

```diff
@@ -0,0 +1,55 @@
+// Copyright 2021 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+
+import 'package:cocoon_service/src/service/datastore.dart';
+import 'package:gcloud/db.dart';
+
+import '../model/appengine/branch.dart';
+import '../model/github/create_event.dart';
+import '../request_handling/exceptions.dart';
+
+class RetryException implements Exception {}
+
+/// Parse a create github webhook event, and add it to datastore.
```

Comment:
This is more a method doc for `handleCreateRequest`. A class doc would mention this handles GitHub branches and manages them in relation to the release process